reladomographql/src/test/resources/test-schema.graphqls

```diff
@@ -0,0 +1,541 @@
+type Mutation
+{
+    testBalanceNoAcmap_insert(testBalanceNoAcmap: TestBalanceNoAcmapInput): TestBalanceNoAcmap
+    order_insert(order: OrderInput): Order
+    product_insert(product: ProductInput): Product
+    orderItem_insert(orderItem: OrderItemInput): OrderItem
+}
+
+
+type Query
+{
+    testBalanceNoAcmapById( accountId: String!  productId: Int!  positionType: Int! ): TestBalanceNoAcmap
+    testBalanceNoAcmaps(findMany: TestBalanceNoAcmapFinder filter: TestBalanceNoAcmapFinder where: TestBalanceNoAcmapFinder order_by: TestBalanceNoAcmapOrderBy limit: Int): [TestBalanceNoAcmap]
+    testBalanceNoAcmap_aggregate(findMany: TestBalanceNoAcmapFinder filter: TestBalanceNoAcmapFinder where: TestBalanceNoAcmapFinder order_by: TestBalanceNoAcmapOrderBy limit: Int): [TestBalanceNoAcmapAggregate]
+
+    orderById( orderId: Int! ): Order
+    orders(findMany: OrderFinder filter: OrderFinder where: OrderFinder order_by: OrderOrderBy limit: Int): [Order]
+    order_aggregate(findMany: OrderFinder filter: OrderFinder where: OrderFinder order_by: OrderOrderBy limit: Int): [OrderAggregate]
+
+    productById( productId: Int! ): Product
+    products(findMany: ProductFinder filter: ProductFinder where: ProductFinder order_by: ProductOrderBy limit: Int): [Product]
+    product_aggregate(findMany: ProductFinder filter: ProductFinder where: ProductFinder order_by: ProductOrderBy limit: Int): [ProductAggregate]
+
+    orderItemById( id: Int! ): OrderItem
+    orderItems(findMany: OrderItemFinder filter: OrderItemFinder where: OrderItemFinder order_by: OrderItemOrderBy limit: Int): [OrderItem]
+    orderItem_aggregate(findMany: OrderItemFinder filter: OrderItemFinder where: OrderItemFinder order_by: OrderItemOrderBy limit: Int): [OrderItemAggregate]
+
+    tamsMithraAccountById( accountNumber: String! ): TamsMithraAccount
+    tamsMithraAccounts(findMany: TamsMithraAccountFinder filter: TamsMithraAccountFinder where: TamsMithraAccountFinder order_by: TamsMithraAccountOrderBy limit: Int): [TamsMithraAccount]
+    tamsMithraAccount_aggregate(findMany: TamsMithraAccountFinder filter: TamsMithraAccountFinder where: TamsMithraAccountFinder order_by: TamsMithraAccountOrderBy limit: Int): [TamsMithraAccountAggregate]
+
+}
+
+
+#################### com.gs.fw.common.mithra.test.domain.TestBalanceNoAcmap ##################
+type TestBalanceNoAcmap
+{
+    accountId: String
+    productId: Int
+    positionType: Int
+    quantity: Float
+    businessDateFrom: Timestamp
+    businessDateTo: Timestamp
+    processingDateFrom: Timestamp
+    processingDateTo: Timestamp
+    businessDate: Timestamp
+    processingDate: Timestamp
+    account: TamsMithraAccount
+}
+
+type TestBalanceNoAcmapAggregate
+{
+    accountId: String
+    productId: Int
```

Comment:
Could the aggregate types by more terse if the meta schema had aggregate types? Then this field would be declared productId: IntAggregate. IntAggregate would be in the meta schema as:
```graphql
type IntAggregate {
  value: Int! # The actual productId
  max: Int!
  min: Int!
  # etc
}
```

Reply:
I figured out the parameterized fileds. and have enum of aggregation types. I can do:

enum { SUM, AVG, MIN, MAX }

and then query field:
quantity(SUM)
quantity(SUM)

this should work according to the graphQL specs but still need to figure out whether the graphql-java supports it.

Reply:
Even if this works, you may want to avoid parameters deep inside a query like this. The problem you may run into is that the json output can have clashing keys with the same name ("quantity" in this case) because the parameters don't become part of the field name.

Reply:
thank you. I have implemented the originally proposed approach:
type FloatAggregate {
  value: Float # The actual productId
  max: Float
  min: Float
}

https://github.com/vborisoff/reladomo/blob/graphql-real/reladomographql/docs/aggregation.md